style.css

```diff
@@ -126,15 +98,14 @@ body {
 	display: flex;
 	justify-content: center;
 	position: relative;
-	z-index: 0;
 	margin-top: -24rem;
 }
 .header-image {
 	width: 80%;
 	max-height: 50rem;
 }
 .button-border {
```

Comment:
Space after selector or no space? Pick one and run with it. Tip: we use a space after selector at YW.